### ISSUE TITLE
Why the batch size is 25 but not 64?

### ISSUE THREAD

**Issue by alwc** [keitakurita/torchtext_tutorials#4]
In the first tutorial, we created the `BucketIterator` with this snippet:

```python
train_iter, val_iter = BucketIterator.splits(
        (trn, vld), # we pass in the datasets we want the iterator to draw data from
        batch_sizes=(64, 64), # (trn_batch_size, vld_batch_size)
        device=0, # if you want to use the GPU, specify the GPU number here
        sort_key=lambda x: len(x.comment_text), # the BucketIterator needs to be told what function it should use to group the data.
        sort_within_batch=False,
        repeat=False # we pass repeat=False because we want to wrap this Iterator layer.
)
```

The `batch_sizes` are `(64, 64)`. 

Next, after we created the `BatchWrapper` and called `next(train_dl.__iter__())`, it shows the tensor has size `[torch.LongTensor of size 494x25]`. 

I've tried on my computer as well and I got the same shape too. How come?

**Comment by alwc** [comment]
By the way, in the blog post you wrote "For the test set, we don't want the data to be shuffled. This is why we'll be using a standard Iterator." 

The `BucketIterator` has `shuffle=None` by default. Without defining it in our `train_iter` and `val_iter`, does it mean the training data are not shuffled?

**Comment by alwc** [comment]
I think I've figure it out why it is 25. It's because there are only 25 rows in `train.csv`...

It is 494 for the other axis because there are 494 tokens in the longest sentence (`id=0005300084f90edc` in `train.csv`).
